MTLManagedObjectAdapter.xcodeproj/project.pbxproj

```diff
@@ -58,6 +58,12 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		3F6F6B412538EB6D00F25078 /* macOS-Application.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = "macOS-Application.xcconfig"; sourceTree = "<group>"; };
```

Comment:
Did all the changes in this file get introduced when you updated to recommended project settings?

Reply:
Yeah, the only changes I manually made were listed in the PR description. Everything else like these pbxproj changes were via Xcode's recommended project settings.